src/test/java/com/getindata/kafka/connect/iceberg/sink/TestIcebergUtil.java

```diff
@@ -242,6 +244,38 @@ public void coerceDebeziumTemporalTypesEnabledBehavior(@TempDir Path localWareho
         assertEquals(record.getField("ship_time_zoned"), OffsetTime.parse("20:32:46.821144Z").toLocalTime());
     }
 
+    @Test
+    public void listStructSchemaHandling()
+      throws JsonProcessingException {
+        IcebergChangeEvent e = new IcebergChangeEvent("test",
+                                          MAPPER.readTree(debeziumMetadataSchema).get("payload"), null,
+                                          MAPPER.readTree(debeziumMetadataSchema).get("schema"), null,
+                                          defaultConfiguration
+        );
```

Comment:
Seems like this is the style round these parts:

```suggestion
        IcebergChangeEvent e = new IcebergChangeEvent("test",
                MAPPER.readTree(debeziumMetadataSchema).get("payload"), null,
                MAPPER.readTree(debeziumMetadataSchema).get("schema"), null,
                defaultConfiguration
        );
```

Reply:
Definitely doesn't use Emacs java mode defaults 😁